schemas/session-replay/mobile/text-style-schema.json

```diff
@@ -6,24 +6,23 @@
   "description": "Schema of all properties of a TextStyle.",
   "allOf": [
     {
-      "required": ["color", "family", "size", "type"],
+      "required": [
+        "color",
+        "family",
+        "size",
+        "type"
```

Comment:
There's no `type` now
```suggestion
```

Reply:
Works for me, for now I leverage the size, family and color so there is no imapct on my side